projects/cashmere/src/lib/typeahead/typeahead-item/typeahead-item.component.ts

```diff
@@ -1,21 +1,34 @@
-import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Component, EventEmitter, Input, Output, Host} from '@angular/core';
+import {TypeaheadComponent} from '../typeahead.component';
 
 @Component({
     selector: 'hc-typeahead-item',
     templateUrl: './typeahead-item.component.html',
-    styleUrls: ['./typeahead-item.component.scss']
+    styleUrls: ['./typeahead-item.component.scss'],
```

Comment:
I think the linter will hate the trailing comma.

Reply:
Change complete!